modules/code-builder/src/brick/design0/BrickBlock.ts

```diff
@@ -52,6 +52,55 @@ export default class BrickBlock extends BrickModelBlock {
     }
 
     public get SVG(): string {
-        return '';
+        const argsLength = Object.keys(this._args).length;
+        const repeatPath = '21 '.repeat(argsLength - 1);
+
+        return `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+                <svg
+                width="114"
+                height="129"
+                version="1.1"
+                id="svg${this._uuid}"
```

Comment:
Should we insert ids for the different tags (i.e. svg, defs, g, g, text, etc.) like this or do you have some other thoughts @meganindya ?

Reply:
ignore IDs for now

Reply:
On it